sr_vision_mocks/README.md

```diff
@@ -0,0 +1,30 @@
+# sr_vision_mocks
+
+This package provides various utilities for mocking vision solutions used at Shadow.
+
+## mock_recognized_objects
+
+This node finds the tfs of recognized objects and publishes them in a proper format to `recognized_objects` topic. That topic is used by grasping pipeline defined in [sr_manipulation_grasp_conductor](https://github.com/shadow-robot/sr_manipulation/tree/F_%23SRC-1550_isee_demo/sr_manipulation_grasp_conductor/src/sr_manipulation_grasp_conductor) package.
+ 
```

Comment:
F_%23SRC-1550_isee_demo change to kinetic-devel?

Reply:
fixed in b3bfa0a